lib/cdo/delete_accounts_helper.rb

```diff
@@ -25,34 +25,17 @@ def initialize(solr: nil)
   # Deletes all project-backed progress associated with a user.
   # @param [Integer] user_id The user to delete the project-backed progress of.
   def delete_project_backed_progress(user_id)
-    # Query the DB for the user's storage ID.
-    user_storage_ids_row = @pegasus_db[:user_storage_ids].where(user_id: user_id).first
-    return unless user_storage_ids_row
-    storage_id = user_storage_ids_row[:id]
-
-    # Delete project data stored on AWS s3.
-    s3_client = AWS::S3.create_client
-    {
-      CDO.animations_s3_bucket => CDO.animations_s3_directory,
-      CDO.assets_s3_bucket => CDO.assets_s3_directory,
-      CDO.files_s3_bucket => CDO.files_s3_directory,
-      CDO.sources_s3_bucket => CDO.sources_s3_directory
-    }.each do |bucket, directory|
-      unless bucket.present? && directory.present?
-        raise "Missing AWS s3 bucket information (bucket: #{bucket}, directory: #{directory})."
-      end
-      s3_client.buckets[bucket].objects.with_prefix("/#{directory}/#{storage_id}/").delete_all
```

Comment:
This code was untested and had apparently never been run - this isn't how the S3 API works at all, it just throws a runtime error at `s3_client.buckets`.  We _do_ need to reimplement this behavior, but for now I'm going to tear it out so we're closer to a functioning state, and I'll put it back in with tests soon.

Reply:
best to also remove https://github.com/code-dot-org/code-dot-org/blob/34121e5141a853dea60ebf5c88962aeaffd0a807/bin/cron/delete_accounts, which references this. I don't see anywhere delete_accounts is being run nor was ever run in the past.

Reply:
Good point - I'll do that in a separate PR.